subiquity/models/ubuntu_pro.py

```diff
@@ -38,11 +38,8 @@ def make_cloudconfig(self) -> dict:
         """
         if not self.token:
             return {}
-        # Both "ubuntu_advantage" and "ubuntu-advantage" keys are accepted, but
-        # "ubuntu-advantage" is deprecated despite not being mentioned in the
-        # documentation.
         return {
-            "ubuntu_advantage": {
+            "ubuntu_pro": {
```

Comment:
@ogayot looks good to me.   Moving to `ubuntu_pro` is supported on 20.04 and later for cloud-init version 24.1 which published to the archive in June.


There are two uploads under review that will make this valid as well for 16.04 and 18.04 if that becomes something you need to support, but that hasn't landed yet.